src/System.IO/ref/System.IO.cs

```diff
@@ -164,6 +164,11 @@ public partial class StreamReader : System.IO.TextReader
         public StreamReader(System.IO.Stream stream, System.Text.Encoding encoding, bool detectEncodingFromByteOrderMarks) { }
         public StreamReader(System.IO.Stream stream, System.Text.Encoding encoding, bool detectEncodingFromByteOrderMarks, int bufferSize) { }
         public StreamReader(System.IO.Stream stream, System.Text.Encoding encoding, bool detectEncodingFromByteOrderMarks, int bufferSize, bool leaveOpen) { }
+        public StreamReader(string path) { }
+        public StreamReader(string path, bool detectEncodingFromByteOrderMarks) { }
+        public StreamReader(string path, System.Text.Encoding encoding) { }
+        public StreamReader(string path, System.Text.Encoding encoding, bool detectEncodingFromByteOrderMarks) { }
+        public StreamReader(string path, System.Text.Encoding encoding, bool detectEncodingFromByteOrderMarks, int bufferSize) { }
```

Comment:
Thanks folks. I'll take these next steps then:

- Close this PR
- Raise issue to bring StreamReader and StreamWriter to CoreFX
- In implementation of the above, use reflection to File.Open [like is done in Environment.WorkingSet](https://github.com/dotnet/corefx/blob/9348997920c1ba782b45adb5e8ee9754149ab1e2/src/System.Runtime.Extensions/src/System/Environment.cs#L227). 
- Include the tests from this PR in the new PR. 
- Add comment to merge the System.IO and System.IO.FileSystem assemblies to #11151